examples/gatekeeper/duplicate-s3/README.md

```diff
@@ -1,11 +1,12 @@
 ### Block Claim creation that reuses vpcName already managed by another Claim
 
-This example Gatekeeper policy denies requests for claims with a vpcName already used by another claim, unless they are observe only
+This example Gatekeeper policy denies requests for Bucket claims or Bucket MRs, if
+there is already a Bucket MR managing the bucket.
 
 Examples and test cases are available under the `samples` directory. Tests can be ran using the [gator cli](https://open-policy-agent.github.io/gatekeeper/website/docs/gator/).
 
 To run tests for this example run:
 ```bash
-cd examples/gatekeeper/unique-names/
+cd examples/gatekeeper/duplicate-s3/
 gator verify . -v
 ```
```

Comment:
Can you expand this with a bit more explanation of what is happening. The `template.yaml` contains the rego policy, the `sample` directory the buckets and `suite.yaml` the test cases... 

Also, how would I apply this to the cluster? Prerequisites, have Gatekeeper installed, apply the template.yaml and then the files in the `sample` directory. And then where do I see the exceptions?